sig-meeting-improvement/README.md

```diff
@@ -0,0 +1,50 @@
+# SIG Meeting Improvement Initiative
+
+## What are the problems we are experiencing?
+- We have too many meetings which employees are expected to attend.
```

Comment:
BTW I have always found challenging the fact of enforcing to have ambassadors from all the teams and having less meetings. SIGs is something that grow, but team size used to be fixed

Reply:
I agree in general. We need to see how we can find a good balance :)